README.md

```diff
@@ -1086,14 +1086,16 @@ A promise represents the eventual result of an asynchronous operation. jQuery ha
   ```js
   // jQuery
   function asyncFunc() {
-    const defer = new $.Deferred();
-    setTimeout(() => {
-      if(true) {
-        defer.resolve('some_value_compute_asynchronously');
-      } else {
-        defer.reject('failed');
-      }
-    }, 1000);
+    const defer = new $.Deferred((deferred) => {
+      setTimeout(() => {
+        if(true) {
+          deferred.resolve('some_value_computed_asynchronously');
+        } else {
+          deferred.reject('failed');
+        }
+      }, 1000);
+    });
+
```

Comment:
I prefer the previous style of using `$.Deferred`, as the example in this jQuery document http://api.jquery.com/deferred.promise/